metadata-ingestion/src/datahub/ingestion/api/committable.py

```diff
@@ -5,9 +5,9 @@
 
 
 class CommitPolicy(Enum):
-    ALWAYS = auto
```

Comment:
@rslanka I think this fixes bug in the current code, it will cause:
```
CommitPolicy.ALWAYS == CommitPolicy.ON_NO_ERRORS == CommitPolicy. ON_NO_ERRORS_AND_NO_WARNINGS == True
```

Reply:
this has been fixed by https://github.com/linkedin/datahub/pull/4092